langchain/vectorstores/utils.py

```diff
@@ -0,0 +1,38 @@
+"""Utility functions for working with vectors and vectorstores."""
+
+from typing import List
+
+import numpy as np
+
+
+def cosine_similarity(a: np.ndarray, b: np.ndarray) -> float:
```

Comment:
should we make this a method of a class?

Reply:
with the argument being it makes it easier for other subclasses to use them?

i dont feel super strongly, i think given that these dont rely on anything in the class and would be static methods anyways, its slightly more modular to keep them separate, but happy to change in future if it starts making things easier